smithy-diff/src/test/java/software/amazon/smithy/diff/evaluators/AddedEntityBindingTest.java

```diff
@@ -53,7 +53,7 @@ public void detectsAddedOperationToResource() {
         Model modelB = Model.assembler().addShapes(r1, o).assemble().unwrap();
         List<ValidationEvent> events = ModelDiff.compare(modelA, modelB);
 
-        assertThat(TestHelper.findEvents(events, "AddedOperationBinding").size(), equalTo(1));
+        assertThat(TestHelper.findEvents(events, "AddedOperationBinding.ToResourceAdded.foo.baz#Operation").size(), equalTo(1));
```

Comment:
Can we just include the shape ID shape name rather than the full ID? A service can only have a single resource/operation in the entire closure with a given shape name, so the full ID actually isn't giving any more detail that we'd need to make these unique.